modules/get-started/pages/whats-new.adoc

```diff
@@ -6,11 +6,15 @@ This topic includes new content added from version 4.29.0 onwards.
 
 For a full list of product updates, and details of updates prior to version 4.29.0, see the https://github.com/redpanda-data/connect/releases[Redpanda Connect release notes^]. For new Cloud-specific features, see xref:redpanda-cloud:get-started:whats-new-cloud.adoc[].
 
+
+== Version 4.44.0
+
+This release reintroduces trial licenses for Self-Managed Redpanda Connect.
```

Comment:
```suggestion
This release introduces support for trial licenses in Redpanda Connect Self-Managed.
```

Reply:
I don't think RPCN ever introduced support for trial licenses.

Reply:
Updated